src/client/components/app/DetailCard/index.tsx

```diff
@@ -45,22 +57,25 @@ const StyledCardContent = styled(CardContent)<{ wrap?: boolean; pointer?: boolea
   &:hover {
     background-color: ${({ theme }) => theme.colors.selectionBar};
 
-    ${StyledCardElement} {
-      color: ${({ theme }) => theme.colors.onSurfaceH2Hover};
-    }
-    .action-button {
-      background: ${({ theme }) => theme.colors.vaultActionButton.selected.background};
-      color: ${({ theme }) => theme.colors.vaultActionButton.selected.color};
-      border: 2px solid ${({ theme }) => theme.colors.vaultActionButton.selected.borderColor};
-    }
+    // NOTE If you want to change other elements on selection bar hover
+    // ${StyledCardElement} {
+    //   color: ${({ theme }) => theme.colors.titles};
+    // }
+    // .action-button {
+    //   background: ${({ theme }) => theme.colors.vaultActionButton.selected.background};
+    //   color: ${({ theme }) => theme.colors.vaultActionButton.selected.color};
+    //   border: 2px solid ${({ theme }) => theme.colors.vaultActionButton.selected.borderColor};
+    // }
```

Comment:
I would consider removing.

Reply:
I would left it as a note to make it easier to modify in the future, I think this code will be reutilized since when you hover an element it changes the background, and the elements and buttons inside should change color sometimes to be visible